tools/generator/dfg/stm32/stm_device_tree.py

```diff
@@ -390,9 +390,15 @@ def deduplicate_list(l):
                 for mux_ch_position, channel in enumerate(channels):
                     m = mux_channel_regex.match(channel)
                     assert m is not None
+                    channel = int(m.group("channel"))
+
+                    # STM32G0B/C device data is wrong, only max 7 channels are available
+                    if did.family == 'g0' and did.name in ['b0', 'b1', 'c1'] and channel > 7:
+                        continue
+
```

Comment:
Do you perhaps want to patch the CubeMX database instead? There's a mechanism for it, and I'm trying to reduce the amount of magic code.

Reply:
Done. Changes are pushed, but don't appear here. github has a major outage of pull requests and actions right now.